### PR TITLE
Removed Tagged Terms

### PR DESCRIPTION
**Pull request by kholtzman5610**
- Removed TaggedTerms controller

- Removed TaggedTerms model

- Removed TaggedTerms mention from the def self.ids_tagged(tag)in study.rb line 25
   terms=Public::TaggedTerm.terms_for_tag(tag).pluck(:term).uniq

- Testing links on all pages to see if all worked

### CODE REVIEW

**Review by micronix — APPROVED** [review]
Looks good

File: app/models/public/study.rb [hunk 1]
```diff
@@ -22,7 +22,6 @@ class Study < Public::PublicBase
     self.primary_key = 'nct_id'
 
     def self.ids_tagged(tag)
-      terms=Public::TaggedTerm.terms_for_tag(tag).pluck(:term).uniq
       ids=Public::BrowseCondition.ids_for_terms(terms) + Public::BrowseIntervention.ids_for_terms(terms)
```
Comment (crystal-wb) [inline app/models/public/study.rb:25]: You can't just remove that line. This method is being given a tag, that tag is setting the term variable that is being called in this line `ids=Public::BrowseCondition.ids_for_terms(terms) + Public::BrowseIntervention.ids_for_terms(terms)`
This method will throw an error when called.

Check to see if this method is called anywhere. If it is not called then you can remove it. If it is called then we will need to discuss keeping the TaggedTerm model.

File: app/models/public/tagged_term.rb [hunk 2]
```diff
@@ -1,20 +0,0 @@
-module Public
-  class TaggedTerm < Public::PublicBase
-    #  Note:  Read-only All public data is managed by other apps (aact & aact-proj).
-    after_initialize :readonly!
-
```
Comment (crystal-wb) [inline app/models/public/tagged_term.rb:5]: Make sure to check for any calls to the methods in this model. Search for the method names. Note if they are saved to variables.

If there are lots of uses of these methods then we'll want to keep the model. If the methods don't seem to be not used or used very rarely, then we can probably remove the model. Though we will have to fix what was using them.

**Review by crystal-wb — CHANGES_REQUESTED** [review]
I noted an issue with a variable that no longer has value because you removed the line of code that referred to the TaggedTerm model. Please check my comments.

**Comment by kholtzman5610** [comment]
### Update on commit:
- Restored line 25 in ( app/models/public/study.rb )
- Restored TaggedTerms model ( app/models/public/study.rb )

**Review by crystal-wb — APPROVED** [review]
looks good

**Pull request merged**